CONTRIBUTING.md

```diff
@@ -78,20 +103,36 @@ python -m pytest pyqtgraph/examples tests
 
 ### Tox
 
-As PyQtGraph supports a wide array of Qt-bindings, and python versions, we make use of `tox` to test against as many supported configurations as feasible.  With tox installed, simply run `tox` and it will run through all the configurations.  This should be done if there is uncertainty regarding changes working on specific combinations of PyQt bindings and/or python versions.
+As PyQtGraph supports a wide array of Qt-bindings, and python versions, we 
+make use of `tox` to test against as many supported configurations as 
+feasible. With tox installed, simply run `tox` and it will run through all the 
+configurations.  This should be done if there is uncertainty regarding changes 
+working on specific combinations of PyQt bindings and/or python versions.
 
 ### Continuous Integration
 
-For our Continuous Integration, we utilize Github Actions.  Tested configurations are visible on [README](README.md).
+For our Continuous Integration, we utilize GitHub Actions.  Tested 
+configurations are visible on [README](README.md).
 
 ### Benchmarks
 
-( *Still under development* ) To ensure this library is performant, we use [Air Speed Velocity (asv)](https://asv.readthedocs.io/en/stable/) to run benchmarks. For developing on core functions and classes, be aware of any impact your changes have on their speed. To configure and run asv:
+To ensure this library is performant, we use 
+[Air Speed Velocity (asv)](https://asv.readthedocs.io/en/stable/) to run 
+benchmarks. For developing on core functions and classes, be aware of any 
+impact your changes have on their speed. To configure and run asv:
 
 ```bash
-pip install asv
+pip install asv virtualenv
 python setup.py asv_config
+asv config
```

Comment:
Error running this command:
```
(asv-test) PS C:\Users\...\pyqtgraph> asv config
usage: asv [-h] [--verbose] [--config CONFIG] [--version]
           {help,quickstart,machine,setup,run,dev,continuous,find,rm,publish,preview,profile,update,show,compare,check,gh-pages} ...
asv: error: argument {help,quickstart,machine,setup,run,dev,continuous,find,rm,publish,preview,profile,update,show,compare,check,gh-pages}: invalid choice: 'config' (choose from 'help', 'quickstart', 'machine', 'setup', 'run', 'dev', 'continuous', 'find', 'rm', 'publish', 'preview', 'profile', 'update', 'show', 'compare', 'check', 'gh-pages')
```

Reply:
That's what I get for going off memory, pretty sure it should be `asv setup`

Reply:
Do tests require python 3.11? If so, might be worth specifying in the docs
```
(asv-test) PS C:\Users\ntjess\Desktop\git\pyqtgraph> asv setup
· No executable found for python 3.11
· No executable found for python 3.11
· No executable found for python 3.11
```

Reply:
Huh; I'm fairly certain I tested on Python 3.11... haven't seen that error before tho